arm64/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM multiarch/qemu-user-static:aarch64 as qemu
-FROM arm64v8/debian:10.6-slim
```

Comment:
What is the advantage of these changes? Does it make the container smaller or larger?